### PR TITLE
add reserve high rate

### PR DESCRIPTION
**Pull request by ilanDoron**
This PR included 3 PRs

wasn't intended. but at this stage should be fine.
Hope u can merge

PR includes:
1. Add high rate support (aligned with Katalyst) in 
- kyberReserve - add new reserve file
- enhanced steps contract
- update tests for enhanced stepd
- create test file for high rate reserve

2. linting all solidity versions in travis script

3. organize folders with following logic.
- all production deployed code should not be touched (including imported paths)
- all non production stuff can be moved. example: order book reserve folder was moved. Also some interface files that are used by wrappers etc. since we can modify wrapper code.

### CODE REVIEW

**Review by JoaquinLiHua — APPROVED** [review]

**Comment by JoaquinLiHua** [comment]
We would like to setup a reserve on Kyber for Boosted, how do we do that?

**Comment by Anyhowclick** [comment]
@ilanDoron can u dismiss this guy's review?
@JoaquinLiHua Hi there, for token listings, kindly fill up the form [here](https://docs.google.com/forms/d/e/1FAIpQLSdOyz3QZ4SnYJvU_1HnjgqX6ZVBVRf5MchudA-gHqf3I3BsLg/viewform). In the meantime, kindly check out this guide:
https://developer.kyber.network/docs/Reserves-AutomatedPriceReserve/

File: .travis.yml [hunk 1]
```diff
@@ -19,10 +19,12 @@ jobs:
         - TEST_PART: Coverage
 
 install:
-  - npm ci
+  - yarn install
 
 before_script:
   - npx solhint "contracts/sol6/**/*.sol"
+  - npx solhint "contracts/sol6/**/*.sol"
+  - npx solhint "contracts/sol6/**/*.sol"
```
Comment (Anyhowclick) [inline .travis.yml]: Why is there double of these?

**Review by Anyhowclick — APPROVED** [review]
Just a small comment, I think we can merge it then fix it? Or u wanna fix it?

File: .solhint.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "avoid-throw": "error",
     "avoid-tx-origin": "off",
     "check-send-result": "error",
-    "compiler-version": ["error", "0.6.6"],
+    "compiler-version": ["off", "0.6.6"],
```
Comment (ilanDoron) [inline .solhint.json:12]: we have multiple versions. this line doesn't make sense

**Pull request merged**